src/function_source.rs

```diff
@@ -43,18 +44,30 @@ impl Source for FunctionSource {
         let query_json_string = query_to_json_string(&query)
             .map_err(|err| io::Error::new(io::ErrorKind::Other, err))?;
 
-        let query = format!(
+        // Query preparation : the schema and function can't be part of a prepared query, so they need to be escaped by hand
+        // The query expects the following arguments :
+        // $1 : x
+        // $2 : y
+        // $3 : z
+        // $4 : query_json
+        let escaped_schema = self.schema.replace("\"", "\"\"");
```

Comment:
I think using the Postgres escapeIdentifier() function would be better here

Reply:
Let me look into that.

Reply:
Found https://docs.rs/postgres-protocol/0.6.1/postgres_protocol/escape/fn.escape_identifier.html